### DISCUSSION TITLE
Why is my scrollbar ridiculously thin, and not expanding when hovering?

### DISCUSSION THREAD

**Discussion by toymil** [microsoft/terminal#14845] [Q&A]
### Windows Terminal version

1.16.10261.0

### Windows build number

10.0.19044.2486

### Steps to reproduce

Fresh install of wt on fresh win10.

The device has touch screen and 175% hidpi scaling.

### Actual Behavior

Across multiply versions of wt, on one of my machine always had a ridiculously thin scrollbar (not just the appearance, the grab box is equally ridiculously thin), it also does not change appearance (expanding or anything) when hovering with mouse:
![Snipaste_2023-02-09_23-30-41](https://user-images.githubusercontent.com/13289283/217859296-2f0d8b4b-1268-4634-b6c9-fef00f12545b.png)

This behaviour is the same across multiple fresh windows 10 installs _on one particular machine_, so I really have no idea what is causing it.  The only difference seems to be touch screen and hidpi scaling.  
As a comparison, this is the normal, thick, usable scrollbar on another machine:
![Snipaste_2023-02-09_23-44-06](https://user-images.githubusercontent.com/13289283/217864131-1e382cc8-ae3a-416f-a5c6-62bc3b242d2e.png)

**Comment by DHowett** [comment]
_Huh._ We changed how the scrollbar is built in 1.16, but since you mentioned that this happens on one specific machine (thanks for the indepth info about it! Those are great data points...) I'm skeptical that that was the cause.

We've seen sporadic reports of this from time to time. Let me see if I can find another copy on this repo.

**Comment by toymil** [comment]
It's not related to 1.16, the scrollbar has been like this for multiple versions now.

**Comment by carlos-zamora** [comment]
Thanks for the feedback! Since it sounds like this is something inconsistent across devices, I'm going to convert this to a Q&A discussion thread.
